File: query_b24.py
import urllib.request
import json

base_url = "https://altaircom.bitrix24.ru/rest/276/nxv29r67o1qp5hd3/"

endpoints = [
    f"crm.deal.get.json?id=43504",
    f"crm.lead.get.json?id=43504",
    f"crm.company.get.json?id=43504",
    f"crm.contact.get.json?id=43504"
]

results = {}
for ep in endpoints:
    url = base_url + ep
    try:
        req = urllib.request.Request(url)
        with urllib.request.urlopen(req) as response:
            data = json.loads(response.read().decode('utf-8'))
            results[ep.split('.')[1]] = data
    except Exception as e:
        results[ep.split('.')[1]] = str(e)

print(json.dumps(results, indent=2, ensure_ascii=False))

File: query_b24.js
const fs = require('fs');
const https = require('https');

const fetchUrl = (url) => {
    return new Promise((resolve, reject) => {
        https.get(url, (res) => {
            let data = '';
            res.on('data', chunk => data += chunk);
            res.on('end', () => resolve(data));
        }).on('error', reject);
    });
};

async function main() {
    const compId = '43504';
    const url = `https://altaircom.bitrix24.ru/rest/276/nxv29r67o1qp5hd3/crm.company.get.json?id=${compId}`;

    try {
        const data = await fetchUrl(url);
        fs.writeFileSync('company_43504.json', JSON.stringify(JSON.parse(data), null, 2));

        // Also let's try to get activities or deals for this company
        const list_deals_url = `https://altaircom.bitrix24.ru/rest/276/nxv29r67o1qp5hd3/crm.deal.list.json?filter[COMPANY_ID]=${compId}`;
        const d_data = await fetchUrl(list_deals_url);
        fs.writeFileSync('company_43504_deals.json', JSON.stringify(JSON.parse(d_data), null, 2));

        const acts = await fetchUrl(`https://altaircom.bitrix24.ru/rest/276/nxv29r67o1qp5hd3/crm.activity.list.json?filter[OWNER_TYPE_ID]=4&filter[OWNER_ID]=${compId}`);
        fs.writeFileSync('company_43504_activities.json', JSON.stringify(JSON.parse(acts), null, 2));

    } catch (e) {
        console.error(e);
    }
}

main();
